libs/scripts/scripts.js

```diff
@@ -22,7 +22,7 @@ const prodDomains = ['milo.adobe.com'];
 const locales = {
```

Comment:
Be aware that each consumer defines their own locales, those are just the locales for milo.adobe.com.

Reply:
Here a bacom exam

Reply:
Yes, once we merge this PR I will reach out to consumers to review their configs.
My initial ticket was actually for CC, but I want Milo project to have a valid configuration and serve as a blueprint for consumer teams, thus fixing it here. 
Except for a few specific cases, most of the locales config should be the same across clouds.